test/pt2_to_circle_test/test_pt2_to_circle.py

```diff
@@ -76,7 +79,12 @@ def convert_nnmodule_to_pt2(
         #   UserWarning: At pre-dispatch tracing, we assume that any custom op marked with
         #     CompositeImplicitAutograd and have functional schema are safe to not decompose.
         _args, _kwargs = helper.get_args_kwargs(example_inputs)
-        exported = export(model.eval(), args=_args, kwargs=_kwargs)
+        if dynamic_shapes is not None:
+            exported = export(
+                model.eval(), args=_args, kwargs=_kwargs, dynamic_shapes=dynamic_shapes
+            )
+        else:
+            exported = export(model.eval(), args=_args, kwargs=_kwargs)
```

Comment:
(Optional suggestion)

```suggestion
        exported = export(
            model.eval(),
            args=_args,
            kwargs=_kwargs,
            dynamic_shapes=dynamic_shapes is not None,
        )
```

Reply:
dynamic_shape is not bool type. But, just forwarding it to the kwargs of `export()` also would work.